lib/data/replacements.json

```
@@ -213,6 +214,18 @@
       }
     ]
   },
+  "docker-compose": {
+    "description": "Compose has been integrated in the official Docker image.",
```

```suggestion
    "description": "Compose is now part of the official Docker image.",
```

I think this is easier to understand, but check if this is technically accurate. 😉